src/v8runtime/V8Runtime.h

```diff
@@ -85,6 +85,9 @@ class V8Runtime : public facebook::jsi::Runtime {
       size_t length) override;
   facebook::jsi::PropNameID createPropNameIDFromString(
       const facebook::jsi::String &str) override;
+  facebook::jsi::PropNameID createPropNameIDFromSymbol(
```

Comment:
i'm afraid this will introduce build error for react-native < 0.69. maybe we should have multiple variants of V8Runtime.

Reply:
@Kudo Instead of separate build variant, we could add a CPP_FLAG `RN_VERSION` which we can use to exclude `createPropNameIDFromSymbol` at build time.